### DISCUSSION TITLE
Add 4x2 grids with Tertiary

### DISCUSSION THREAD

**Discussion by AadCYH** [chrenn/swish-dl#17] [Ideas]
Hi, I bought Swish in 2019 and have enjoyed it a lot, thanks for the amazing app.

For Swish window snapping, in the past, I often used 2x2 and 3x2 with my MBP.  Recently, I find 4x2 becoming my preference with my ultrawide monitor.

With ultrawide monitor, the left 1/4 is for my referring material, the right 1/4 is for Messages, Teams, Telegram, LINE, and some other communication apps, the center 2/4 is for my workspace.  Sometimes, I will also use the other 4x2 windows combination.  And I don't use 3x3.

So, I would like to recommend to add 4x2 grids and have 4x2 and 3x3 options with Tertiary, user can choose which option for preference.

Is it a good idea? Please reply and let me know, I will be happy to help if any required.

**Comment by chrenn** [comment]
Swish's swiping system only allows for up to a 3x3 grid. Swish 2 (paid upgrade coming later this year) will feature custom grids via dragging.
